### DISCUSSION TITLE
Need field “availableCryptoCodes” on “Get Store” response

### DISCUSSION THREAD

**Discussion by woutersamaey** [btcpayserver/btcpayserver#3192] [Technical Support]
Can we get a field like this? Or is there another way to get a list of all supported cryptoCode options for a given store id?

**Comment by bolatovumar** [comment]
@woutersamaey there is `/api/v1/stores/{storeId}/payment-methods` that exists. I guess we could also return it from `/api/v1/stores/{storeId}` but maybe there is a reason we wouldn't want to.

**Reply by woutersamaey** [reply]
I've proposed a solution in #3232 that would solve this better. Basically a new API endpoint for getting all cryptoCodes and currencyCodes.
If you want to help, you're very welcome or I'll propose a PR when I'm ready.

FYI I'm talking about the 1st call in the swagger I drafted.
